### ISSUE TITLE
Binder ignores invalid value in DatePicker

### ISSUE THREAD

**Issue by tepi** [vaadin/flow-components#1268]
- Description of the bug
    - DatePicker which is bound to a binder and has an invalid value entered by the user ends up with the binder status being valid.
- Minimal reproducible example
```
Binder<Beanie> binder = new Binder<>();
binder.forField(dp).withConverter(new LocalDateToDateConverter())
		.bind(Beanie::getDate, Beanie::setDate);
binder.setBean(new Beanie("Normal"));
binder.setValidationStatusHandler(e -> {
	System.err.println("Status: " + e.isOk());
	e.getFieldValidationErrors()
			.forEach(err -> System.err.println("Error: " + err.getMessage().orElse("n/a")));
	e.getBeanValidationErrors().forEach(err -> System.err.println("Error: " + err.getErrorMessage()));
});
```
- Expected behavior
    - When the user enters "foobar" into the DatePicker in the browser, the DatePicker is highlighted with red background which is expected. However when the binder's ValidationStatusHandler is triggered, the printouts in the code report that everything is ok and there are no errors.

![Screen Shot 2019-04-08 at 10 56 33](https://user-images.githubusercontent.com/1527900/55707755-0820cf80-59ed-11e9-854b-47060cd336f4.png)

- Actual behavior
    - Binder validation should not pass since there clearly is an invalid value in the DatePicker.
- Versions:
    - Vaadin / Flow version: 10.0.12
    - Java version: 1.8.0_121.jdk
    - OS version: macOS Sierra 10.12.6
    - Browser version: Version 73.0.3683.103 (Official Build) (64-bit)

**Comment by vursen** [comment]
We have just introduced new validation behavior aiming to solve many different validation issues that people were complaining about, including this one. The new behavior is available under the feature flag in v23.2 for several components including DatePicker.

To enable the feature flag in your application, add the following to `vaadin-featureflags.properties`:

```
com.vaadin.experimental.enforceFieldValidation=true
```

Any feedback would be highly appreciated. For more details, please refer to https://github.com/vaadin/platform/issues/3066.

**Comment by vursen** [comment]
The issue has been resolved by the [new validation mechanism](https://github.com/vaadin/platform/issues/3066) which is enabled by default in Vaadin 24. For any further bugs, please create separate issues.
